packages/core/util/Job.js

```diff
@@ -114,6 +114,7 @@ class Job {
 	 */
 	throttleUntil = (timeout, ...args) => {
 		if (!this.id) {
+			this.type = 'throttle';
```

Comment:
Let's use `timeout` instead of `throttle` since the primary purpose is to know what kind of "stop" to use. I'll make that change and move this along.